src/RA_Defs.h

```diff
@@ -87,26 +87,24 @@
 
 
 
-
-//	RA-Only
-
-
-//	RA-Only
-#define RAPIDJSON_HAS_STDSTRING 1
-#pragma warning(push, 1)
-// This is not needed the most recent version
-#define _SILENCE_ALL_CXX17_DEPRECATION_WARNINGS
 //	RA-Only
+#pragma warning(push)
+// Seems version 1.1.0 still uses deprecated stuff.
+#pragma warning(disable : 4996) // STL4015 deprecation: uses iterator as a base class instead of usings from iterator_traits.
```

Comment:
the documented way to handle this is with `#define RAPIDJSON_NOMEMBERITERATORCLASS`

Reply:
thanks.